### PR TITLE
Combine getRollup with getExplain API in backend and other code fix

### PR DESCRIPTION
**Pull request by leeyun-amzn**
*Issue #, if available: N/A*

*Description of changes:*

- Combine explain API to `getRollup` and `getRollups` in backend, so the response will have metadata included with rollup item

- Delete tests related to explain API

- Data model fix. Deleted duplicate rollup data model "RollupItem" and use "DocumentRollup"

- Change all wordings of "recurring" to "continuous"

- Modify the `concatName() `to `getSelectedRollupIds()`and simplified code for the function in `Rollups.tsx`

- Change the check of field.type whether it is null to whether it is undefined due to the parsing in `getMappings` in `CreateRollupForm.tsx `, the field type is directly assigned by the mappings type.
 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: public/pages/CreateRollup/utils/constants.ts [hunk 1]
```diff
@@ -107,6 +107,24 @@ export const testRollup = {
       },
     ],
   },
+  metadata: {
```
Comment (dbbaughe) [inline public/pages/CreateRollup/utils/constants.ts:110]: Are these used in the source code somewhere or just tests? If only tests they should not be mixed in the source code files.

Comment (leeyun-amzn) [reply public/pages/CreateRollup/utils/constants.ts]: Only used in tests, so I moved these 2 constants to `test/testConstants.ts` file.

Comment (leeyun-amzn) [reply public/pages/CreateRollup/utils/constants.ts]: Renamed to `test/constants.ts`

File: models/interfaces.ts [hunk 2]
```diff
@@ -112,17 +85,18 @@ export interface Rollup {
...
   enabledTime: number | null;
-  lastUpdatedTime: number;
-  metadata_id: number | null;
+  last_updated_time: number;
+  metadata_id: string | null;
   metrics: MetricItem[];
   page_size: number;
   schedule: IntervalSchedule | CronSchedule;
-  schemaVersion: number;
+  schema_version: number;
```
Comment (dbbaughe) [inline models/interfaces.ts:96]: If these are used on the frontend we want them to be camelCase not snake_case. If it's a big change let's create a github issue to track the work and we can do it as a follow up PR.

**Review by dbbaughe — APPROVED** [review]

**Review by bowenlan-amzn — APPROVED** [review]

**Pull request merged**